streaming/src/main/scala/io/gearpump/streaming/task/TaskActor.scala

```diff
@@ -151,7 +151,7 @@ class TaskActor(
     var done = false
 
     var count = 0
-    val start = System.currentTimeMillis()
+    val start = System.nanoTime()
```

Comment:
The variable name doesn't contain unit info. Usually gearpump uses millis as *default* unit, but here it is nanos. Maybe add a suffix?